unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarMeshCombiner/AvatarMeshCombinerHelper.cs

```diff
@@ -37,7 +34,7 @@ public AvatarMeshCombinerHelper (GameObject container = null)
         /// <param name="renderersToCombine">A list of avatar parts to be combined</param>
         /// <param name="materialAsset">A material asset that will serve as the base of the combine result. A new materialAsset will be created for each combined sub-mesh.</param>
         /// <returns>true if succeeded, false if not</returns>
-        public bool Combine(SkinnedMeshRenderer bonesContainer, SkinnedMeshRenderer[] renderersToCombine, Material materialAsset)
+        public SkinnedMeshRenderer Combine(SkinnedMeshRenderer bonesContainer, SkinnedMeshRenderer[] renderersToCombine, Material materialAsset)
```

Comment:
Why do we need this change? You can always access `avatarCombinerHelper.renderer`.

Reply:
Yeap, I never saw that property, now I changed back the class to be as it was.